packages/client/lib/index.js

```diff
@@ -1,15 +1,72 @@
+const shuffle = require('lodash.shuffle')
 const HttpClient = require('./http')
 const resources = require('./resources')
+const initialPeers = require('./peers')
+const sortPeers = require('./utils/sort-peers')
 
 module.exports = class ApiClient {
+  /**
+   * Finds all the available peers, sorted by block heigh and delay
+   *
+   * @param {String} network - Network name
+   * @param {Number} version - API version
+   */
+  static async findPeers (network, version) {
+    if (!initialPeers.hasOwnProperty(network)) {
+      throw new Error(`Network "${network}" is not supported`)
+    }
+
+    const networkPeers = initialPeers[network]
+    let peers = null
+
+    // Shuffle the peers to avoid connecting always to the first ones
+    shuffle(networkPeers)
+
+    // Connect to each peer to get an updated list of peers until a success response
+    for (let i = 0; i < networkPeers.length; i++) {
+      const peer = networkPeers[i]
+
+      const client = new ApiClient(peer.ip, version)
+      const response = await client.resource('peers').all()
+      const { data } = response.data
+
+      if (data.success && data.peers) {
+        peers = data.peers
```

Comment:
This may not be the best way to do it. In theory (albeit difficult to do), I could gain access to one of the nodes in `peers.js` and return a different set of peers here. As I say, it'd be very difficult to do but not impossible. It can always be revised later though if needed, as it wouldn't always cause issues anyway due to the shuffling

Reply:
Yes, I expect that in the next weeks we start to sign the seed peers.